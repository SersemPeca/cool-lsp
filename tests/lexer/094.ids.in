foo.bar.baz.qux
1startswithnumber
contains-dash
Type-Contains-Dash
_cannot_start_with_underscore
Type-object
