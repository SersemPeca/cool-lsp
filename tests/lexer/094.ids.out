#1 OBJECTID foo
#1 '.'
#1 OBJECTID bar
#1 '.'
#1 OBJECTID baz
#1 '.'
#1 OBJECTID qux
#2 INT_CONST 1
#2 OBJECTID startswithnumber
#3 OBJECTID contains
#3 '-'
#3 OBJECTID dash
#4 TYPEID Type
#4 '-'
#4 TYPEID Contains
#4 '-'
#4 TYPEID Dash
#5 ERROR: Invalid symbol "_"
#5 OBJECTID cannot_start_with_underscore
#6 TYPEID Type
#6 '-'
#6 OBJECTID object
